commands/01-mod.js

```diff
@@ -733,8 +733,16 @@ Module
       if (num > 0) {
         while (num > 0) {
           let deleting = Math.min(num, 50)
-          await channel.bulkDelete(deleting);
-          num -= deleting;
+          deleted = await channel.bulkDelete(deleting, filterOld=true);
```

Comment:
js isn't really pythonic in this way. You can't set `varname = value` in a function call.
```suggestion
          deleted = await channel.bulkDelete(deleting, true);
```